opencog/ghost/README.md

```diff
@@ -109,3 +111,76 @@ One can quickly test if a rule can be triggered by using `test-ghost`:
 ```
 
 The output `[INFO] [Ghost] Say: "Hello human"` will be printed.
+
+## To Run With ECAN (experimental)
+1) Start the [RelEx server](https://github.com/opencog/relex#opencog-serversh)
+2) Start the CogServer, e.g.
+
+```
+opencog/build/opencog/cogserver/server/cogserver
+```
+
+3) Connect to the CogServer in a new terminal, e.g.
+
+```
+telnet localhost 17001
+```
+
+4) Load the ECAN module
+
+```
+loadmodule opencog/build/opencog/attention/libattention.so
+```
+
+5) Start ECAN
+
+```
+start-ecan
+```
+
+6) Load the needed modules
+
+```
+(use-modules (opencog)
+             (opencog nlp relex2logic)
+             (opencog openpsi)
+             (opencog attention)
+             (opencog eva-behavior)
+             (opencog ghost))
+```
+
+7) Before creating any rules, run
+
+```
+(ecan-based-ghost-rules #t)
+```
+
+Note, rules being created after running this will be slimmer (preferred) and can only work with this ECAN approach. They are NOT backward-compatible with the `test-ghost`.
```

Comment:
what does `slimmer` mean? And why two types of rules?

Reply:
It has fewer atoms, the ones that are for `DualLink` are not needed for this ECAN approach